1.10/installing/custom/configuration/configuration-parameters.md

```diff
@@ -501,3 +501,29 @@ Indicates whether to enable the DC/OS proxy.
 For more information, see the [examples](/docs/1.10/installing/custom/configuration/examples/#http-proxy).
 
 **Important:** You should also configure an HTTP proxy for [Docker](https://docs.docker.com/engine/admin/systemd/#/http-proxy).
+
+### <a id="zk-superuser"></a> zk_super_credentials (Enterprise DC/OS Only)
```

Comment:
FTR I am just working on the assumption that these links are done in the right way.